cocos/2d/CCScene.h

```diff
@@ -109,7 +109,7 @@ class CC_DLL Scene : public Node
      * @param renderer The renderer use to render the scene.
      * @js NA
      */
-    virtual void render(Renderer* renderer, const Mat4& eyeTransform, const Mat4* eyeProjection = nullptr);
+    virtual void render(Renderer* renderer, const Mat4* eyeTransforms = nullptr, const Mat4* eyeProjections = nullptr, unsigned int mutiViewCount = 1);
```

Comment:
It is better to add a new function 
```c++
virtual void render(Renderer* renderer, const Mat4* eyeTransforms = nullptr, const Mat4* eyeProjections = nullptr, unsigned int multiViewCount)
```

And there is an typo, `mutiViewCount` -> `multiViewCount`